lms/djangoapps/badges/utils.py

```diff
@@ -0,0 +1,26 @@
+"""
+Utility functions used by the badging app.
+"""
+from django.conf import settings
+
+
+def site_prefix():
+    """
+    Get the prefix for the site URL-- protocol and server name.
+    """
+    scheme = u"https" if settings.HTTPS == "on" else u"http"
+    return u'{}://{}'.format(scheme, settings.SITE_NAME)
+
+
+def requires_badges_enabled(function):
+    """
+    Decorator that bails a function out early if badges aren't enabled.
+    """
+    def wrapped(*args, **kwargs):
+        """
+        Wrapped function which bails out early if bagdes aren't enabled.
+        """
+        if not settings.FEATURES.get('ENABLE_OPENBADGES', False):
```

Comment:
I think I noted somewhere above, but would be good to hide this behind an 'is_feature_enabled' helper method.